src/forms.rs

```diff
@@ -76,8 +77,211 @@ impl<'a> Field<'a> {
     }
 }
 
+/// Only permissible on text fields.
+impl<'a> Field<'a> {
+    // TODO: the spec likely means the equivalent of unicode graphemes here
+    //       for characters
+
+    /// Write the `/MaxLen` attribute to set the maximum length of the field's
+    /// text in characters. Only permissible on text fields.
+    pub fn text_max_len(&mut self, len: i32) -> &mut Self {
+        self.dict.pair(Name(b"MaxLen"), len);
+        self
+    }
+
+    /// Write the `/V` attribute to set the value of this text field.
+    /// Only permissible on text fields.
+    pub fn text_value(&mut self, value: TextStr) -> &mut Self {
+        self.dict.pair(Name(b"V"), value);
+        self
+    }
+
+    /// Start writing the `/DV` attribute to set the default value of this text
+    /// field. Only permissible on text fields.
+    pub fn text_default_value(&mut self, value: TextStr) -> &mut Self {
+        self.dict.pair(Name(b"DV"), value);
+        self
+    }
```

Comment:
Where is `DV` defined for text fields?

Reply:
12.7.5.3 Reset-Form Action (linked to from Section 12.7.3.1 (Table 220, entry `/DV`)) it says that the value in `/DV` is the value `/V` reverts to when a `Reset-Form` action is done. Therefore, it should be available on all fields in the same format as `/V` is.